mlir/test/Dialect/SparseTensor/no_lowering.mlir

```diff
@@ -0,0 +1,52 @@
+// RUN: mlir-opt %s --lower-sparse-ops-to-foreach --split-input-file | FileCheck %s
+
```

Comment:
perhaps add a top level comment on the specific purpose of the no lowering test

Reply:
Done